package.json

```diff
@@ -65,6 +68,16 @@
                 "peerDependencies": {
                     "@types/react": "*"
                 }
+            },
+            "@types/hoist-non-react-statics": {
+                "peerDependencies": {
+                    "@types/react": "*"
+                }
+            },
+            "@types/react-transition-group": {
+                "peerDependencies": {
+                    "@types/react": "^17.0.0"
```

Comment:
why is it set types/react 17 and not react 18?

Reply:
This is for a package used in Admin, which is using React 17.